### PR TITLE
Validate researcher url name is not longer than 350 characters

### PR DESCRIPTION
**Pull request by amontenegro**

### CODE REVIEW

**Comment by amontenegro** [comment]
https://trello.com/c/6jCZnGKk/2381-fix-error-on-adding-researcher-urls

**Comment by rcpeters** [comment]
Let's make it longer
http://www.boutell.com/newfaq/misc/urllength.html 2000.

**Comment by amontenegro** [comment]
@rcpeters ok!

File: orcid-web/src/main/java/org/orcid/frontend/web/controllers/BaseController.java [hunk 1]
```diff
@@ -666,6 +666,10 @@ protected void validateBiography(Text text) {
     }
 
     protected void validateUrl(Text url) {
+        validateUrl(url, 350);
```
Comment (rcpeters) [inline orcid-web/src/main/java/org/orcid/frontend/web/controllers/BaseController.java]: 350?

**Comment by amontenegro** [comment]
@rcpeters dont merge yet, will let you know when it is ready

**Comment by amontenegro** [comment]
@rcpeters you can merge it whenever you want!

**Pull request merged**